engine/planner.go

```diff
@@ -76,6 +77,13 @@ func (p *planner) Plan(ctx context.Context) error {
 
 	ctx = logger.WithContext(ctx, log)
 
+	_, err = p.updateBusy(ctx)
```

Comment:
I would recommend only executing this routine if the user has enabled this setting.  This avoids unnecessary database and API calls

```
if p.MinIdle > 0 {
  _, err = p.updateBusy(ctx)
}
```

Reply:
ah yes.  i remember thinking that at one point before falling asleep.  i'll update it.